ui/dashboard/src/pages/organizations/page-loader.tsx

```diff
@@ -1,14 +1,24 @@
 import { useState } from 'react';
+import { Trans } from 'react-i18next';
+import { organizationArchive, organizationUnArchive } from '@api/organization';
```

Comment:
Same for the other places.

```suggestion
import { organizationArchive, organizationUnarchive } from '@api/organization';
```